src/pages/Sketch/components/Canvas.js

```diff
@@ -276,6 +279,13 @@ function Canvas() {
         context.stroke();
     }
 
+    function eraserMove(point){
+        context.lineWidth=`${eraserSize}`;
+        context.strokeStyle="#FFFFFF";
```

Comment:
@shruthii625 This can be temporary solution, but not a perfect solution. Painting with white color is not erasing!

@smaranjitghose @anushbhatia

Reply:
> @shruthii625 This can be temporary solution, but not a perfect solution. Painting with white color is not erasing!
> 
> @smaranjitghose @anushbhatia

The canvas is white as of now and hence I set it to white. If in the future the background color of the canvas becomes a modifiable attribute, I'll just set the context.strokeStyle to that value. Hope that solves it!